gnovm/cmd/gno/bug.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+const bugTmpl = `## [Subject of the issue]
+
+### Description
+
+Describe your issue in as much detail as possible here
+
+### Your environment
+
+* go version {{.GoVersion}} {{.Os}}/{{.Arch}}
+* version of gno
```

Comment:
can we somehow get this information with `debug.BuildInfo`? https://pkg.go.dev/runtime/debug#BuildInfo

otherwise we do it the good ol' way and put the version as a [compile-time injected variable](https://blog.alexellis.io/inject-build-time-vars-golang/) when running `make build` / install.

Reply:
> can we somehow get this information with debug.BuildInfo? https://pkg.go.dev/runtime/debug#BuildInfo

From my understanding of `debug.BuildInfo`, I'm not entirely sure if there's a way.

> otherwise we do it the good ol' way and put the version as a [compile-time injected variable](https://blog.alexellis.io/inject-build-time-vars-golang/) when running make build / install.

I like this idea. In fact, I was already thinking about it. Later, I decided to work on `gno version` instead.

already added as TODO  here -> https://github.com/gnolang/gno/pull/1325/files#diff-2ba8fa8e90302c604c98727b5c82205ad57c1952fda669f9b06db798930fb6d4R85

Reply:
:+1: 

I've taken a look at BuildInfo; I think that by extracting module information, we could get the commit being used; but I didn't delve too much into it. Maybe try playing around with it before committing to the compile-time injected variable, as BuildInfo has the advantage of working even with `go build` and `go install`. :+1: for doing this in another PR, though.

Reply:
Didn't know that. Thanks for the info. 

Done: 802139e6a55038b25361e5c1b78206e42aa92f0c